### DISCUSSION TITLE
Page not accessible any more because of pipe-character in Path

### DISCUSSION THREAD

**Discussion by Congenial-GmbH** [requarks/wiki#6658] [Error / Bug Report]
Hi,

I wanted to move a page into a new directory and by accident I mistyped a pipe character `|` instead of a forward slash `/` for the new path:

- Incorrect path: `https://example.net/en/MyTopic/SubTopic|MyPageName`

- Correct path: `https://example.net/en/MyTopic/SubTopic/MyPageName`

If I want to access my page now (by the incorrect path),  it says, that this page does not exist. ("This page does not exist yet. Would you like to create it now?").

How can I access my page again and correct its path? 

I report this as a bug, because pipe-characters should not be allowed to be entered.

**Comment by fnafox** [comment]
It should get sanitized. It was probably created with the path `https://example.net/en/MyTopic/SubTopicMyPageName` so just move it accordingly. If you check recent pages on your admin dashboard you should see it there.

**Reply by Congenial-GmbH** [reply]
Thanks for you reply.
No, the issue is exactly, that it was not sanitized. Clicking on the caption in the browse-tree opens page `https://example.net/en/MyTopic/SubTopic|MyPageName` with the corresponding error message, that the site does not exist.

If I manually change the link to `https://example.net/en/MyTopic/SubTopic/MyPageName` , and press <kbd>Enter</kbd> to reload the page. The page reloads, but it says again, that the page does not exist. URL is still displayed as `https://example.net/en/MyTopic/SubTopic/MyPageName` in the browser

**Reply by P4mes4n** [reply]
I just replicated the issue and it seems the page is just lost. You cant access and move it back, not even from the admin panel.
The only way I could see is directly accessing it in the DB and moving it back to an accessible path.

**Reply by Congenial-GmbH** [reply]
Thank you @P4mes4n. What are the requred steps to access it in the DB and moving it back to an accessible path?

**Reply by P4mes4n** [reply]
Unfortunately I don't know. You could probably find out by just digging into the DB. I currently don't have the time to do so.

**Reply by simonclayton** [reply]
I just discovered this issue that "Move / Rename" doesn't sanitise the input and I accidentally had a hash in my path which broke the page in the same way. The way I fixed it was to connect to Postgres and run a SQL query to update the path field in the pages table for the id of the page. I then had to rebuild the tree and it was ok.
